src/test/java/com/epam/training/student_gregory_kubya/MainPage.java

```diff
@@ -1,13 +1,57 @@
 package com.epam.training.student_gregory_kubya;
+
+import java.time.Duration;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
 
-// page_url =
 public class MainPage {
 
-    public MainPage(WebDriver driver) {
-        PageFactory.initElements(driver, this);
-    }
+  private final WebDriver driver;
+  private static final String pageURL = "https://fiftyoutlet.com/";
```

Comment:
Pls use name per Java name convention:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

this error should be highlighted by IDEA